src/browser/modules/Editor/Editor.jsx

```diff
@@ -143,18 +134,13 @@ export class Editor extends Component {
     const onlyWhitespace = cmd.trim() === ''
 
     if (!onlyWhitespace) {
-      this.props.bus.send(
-        EXECUTE_COMMAND_ORIGIN,
-        EXECUTE_COMMAND_ORIGINS.EDITOR
-      )
-      this.execCommand(cmd)
+      this.props.runCommand()
```

Comment:
what does the `runCommand` do? Also, I assume with changes in this work the `EXECUTE_COMMAND_ORIGIN` stuff from yesterday is no longer needed? Ill pull the branch again in a bit and try it